lib/graphql/schema/directive/one_of.rb

```diff
@@ -6,6 +6,18 @@ class OneOf < GraphQL::Schema::Directive
         description "Requires that exactly one field must be supplied and that field must not be `null`."
         locations(GraphQL::Schema::Directive::INPUT_OBJECT)
         default_directive true
+
+        def initialize(...)
```

Comment:
The gem now supports Ruby 2.7+, so this should work. Feel free to change it to something else if you prefer.